spotifycli/spotifycli.py

```diff
@@ -334,9 +337,9 @@ def show_position():
     metadata = get_spotify_property("Metadata")
     position_raw = get_spotify_property("Position")
     # Both values are in microseconds
-    position = datetime.timedelta(milliseconds=position_raw / 1000)
+    position = datetime.timedelta(milliseconds=int(position_raw) / 1000)
     length = datetime.timedelta(
-        milliseconds=metadata['mpris:length'][1] / 1000
+        milliseconds=int(metadata['mpris:length'][1]) / 1000
```

Comment:
And these `int` changes are the other artifacts of dbus message mocking.  Again, I'll provide more detail on this in a following comment, but as it is right now, this function will allow these position values to come in as either string numerics, or actual numerics.  It will `int` them either way, so it will have no effect on existing behavior.